explorer/interpreter/interpreter.cpp

```diff
@@ -1486,26 +1486,23 @@ auto Interpreter::StepExp() -> ErrorOr<Success> {
             ElementPath::Component member(&access.member(), found_in_interface,
                                           witness);
             const Value* aggregate;
-            const Value* me_value;
+            Nonnull<const Value*> me_value = act.results()[0];
```

Comment:
Maybe we should rename this to `lhs_value`, given that it's not the `me` value for a type access (there is no `me` value in that case)? Or make it `optional`.

Reply:
I think it's only accessed for bound members, so switching to an optional seems better for your request.